app/components/NoticeTypeCheckboxes/NoticeTypeCheckboxes.tsx

```diff
@@ -183,11 +183,18 @@ const NoticeTypeLinks: { [key: string]: string | undefined } = {
 }
 
 const JsonNoticeTypes = {
+  IceCube: [
+    'gcn.notices.icecube.GoldAndBronze',
+    'gcn.notices.icecube.LvkNuTrackSearch',
+  ],
   LVK: ['igwn.gwalert'],
+  Swift: ['gcn.notices.swift.bat.guano.Alert'],
 }
 
 const JsonNoticeTypeLinks: { [key: string]: string | undefined } = {
+  IceCube: 'icecube',
```

Comment:
This doesn't look like a valid link to me...

Reply:
Woops, copied from the other link array, fixed in latest commit